scripts/toc.js

```diff
@@ -0,0 +1,9 @@
+const {sync: globSync} = require('glob');
+const {resolve} = require('path');
+const toc = require('markdown-toc');
+const {outputFileSync} = require('fs-extra');
+const {readFileSync} = require('fs');
+globSync(resolve(__dirname, '../docs/*.md')).forEach(fn => {
```

Comment:
if this is run again will output 2 TOC's or is it smart enough to update the existing one?

Reply:
nope it checks for the `<!-- toc -->` thing